state/application.go

```diff
@@ -272,10 +272,6 @@ func (a *Application) removeOps(asserts bson.D) ([]txn.Op, error) {
 			Id:     a.doc.DocID,
 			Assert: asserts,
 			Remove: true,
-		}, {
-			C:      settingsC,
-			Id:     a.settingsKey(),
-			Remove: true,
```

Comment:
how is this redundant?
One is removing the values from applications collection, the other is removing the application-specific settings from the settings collection.
I'm pretty sure we do need both.